### PR TITLE
Feature/#151/withdraw flow

### PR DESCRIPTION
**Pull request by andre-meyer**
# Description

Implements the modals for withdraw flow.

# To Test
- [x] On an active strategy, click the "Withdraw Request (5min)" Button and observe the modal
  - [x] Should look like design: (https://projects.invisionapp.com/share/VXXWLEE7534#/screens/429017378 )
  - [x] Check the checkbox for not being reminded againversion bump)
- [x] On another active strategy, click the "Withdraw Request (5min)" Button and observe no modal appearing.

# Background
`localforage` used. This "withdraw flow skip" info is pinned to the current version (from `package.json`, will need to be re-accepted on next version)

### CODE REVIEW

File: src/components/TabView.tsx [hunk 1]
```diff
@@ -7,19 +7,15 @@ import Deploy from "routes/Deploy";
...
 import { SafeStyleTabHeaderWithCounter } from "./navigation/tabs/SafeStyleTabHeaderWithCounter";
-import { Badge } from "./basic/display/Badge";
 
 const DEFAULT_TAB = "deployment";
 
 export const TabView = memo(function TabView(): JSX.Element {
   const history = useHistory();
 
-  const strategyCount = useRecoilValue(totalStrategyCount);
+  const strategyCount = useRecoilValue(strategyCountByStatus("ACTIVE"));
```
Comment (andre-meyer) [inline src/components/TabView.tsx:18]: This change is unrelated to withdraw flow. It will make the count on the `Strategies` tab only count the active strategies.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 * **🔐 [Safe app link](https://pr167--safecmmappreact.review.gnosisdev.com)**: To be added as a custom app on [Safe Multisig Rinkeby](https://rinkeby.gnosis-safe.io/), [Mainnet](https://gnosis-safe.io/) or [xDai](https://xdai.gnosis-safe.io)
 * **📖 [Storybook](https://pr167--safecmmappreact.review.gnosisdev.com/storybook/)**: Component stories

File: src/components/modal/WithdrawLiquidity/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+// const defaultStates = [
+//   [baseTokenAddressAtom, "0x6B175474E89094C44Da98b954EedeAC495271d0F"],
+//   [quoteTokenAddressAtom, "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"],
+//   [lowestPriceAtom, "300"],
+//   [startPriceAtom, "323"],
+//   [highestPriceAtom, "350"],
+//   [baseTokenAmountAtom, "1000"],
+//   [quoteTokenAmountAtom, "1000"],
+//   [totalBracketsAtom, "10"],
+// ];
```
Comment (alfetopito) [inline src/components/modal/WithdrawLiquidity/index.stories.tsx:19]: Copied too much?

```suggestion
```

File: src/components/modal/WithdrawLiquidity/index.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+      <Header variant="body1">
+        This action will remove all liquidity from your strategy and effectively{" "}
+        <Emphasis>stop it</Emphasis>. Two actions and transactions are required:
+      </Header>
+      <Well>
+        <Steps steps={steps} />
+      </Well>
+      <CheckboxRow>
+        <Checkbox
+          label="Don't show this modal next time"
```
Comment (alfetopito) [inline src/components/modal/WithdrawLiquidity/index.tsx:88]: What about:

```suggestion
          label="Don't show this confirmation next time"
```

File: src/theme.ts [hunk 4]
```diff
@@ -8,6 +8,8 @@ const newColors = {
   backgroundError: "#FFE6EA",
   backgroundWarning: "#FFF3E2",
   backgroundSideBar: "#F7F5F5",
+  backgroundBadgeGray: "#e8e7e6",
+  backgroundBadgeDarkGray: "#b2b5b2",
```
Comment (alfetopito) [inline src/theme.ts:12]: Do I speak "british" without knowing it? https://www.merriam-webster.com/words-at-play/gray-vs-grey-usage-difference

File: src/utils/asPromise.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export const asPromise = (val: any): Promise<any> => {
```
Comment (alfetopito) [inline src/utils/asPromise.ts:1]: Might as well use generic type

(have not tested)

```suggestion
export const asPromise = <T>(val: T): Promise<T> => {
```

**Review by alfetopito — APPROVED** [review]
No major comments, working fine :+1:

File: src/theme.ts (hunk 4 above)
Comment (andre-meyer) [reply src/theme.ts:12]: Seems like it :D
